zephyr/src/indigo_api_callback_dut.c

```diff
@@ -1983,7 +1983,7 @@ static int send_sta_disconnect_handler(struct packet_wrapper *req, struct packet
 static int send_sta_reconnect_handler(struct packet_wrapper *req, struct packet_wrapper *resp) {
     struct wpa_ctrl *w = NULL;
     char *message = TLV_VALUE_WPA_S_RECONNECT_NOT_OK;
-    char buffer[256], response[1024];
+    char buffer[256], response[8];
```

Comment:
This should be a `fixup! [nrf noup] <original issues>`

Reply:
> This should be a `fixup! [nrf noup] <original issues>`

Changes are part of the initial commit in which zephyr support was added and the commit got merged. so not using any fixup sauce tag.